### ISSUE TITLE
log_configuration block for aws_fsx_lustre_file_system

### ISSUE THREAD

**Issue by jesseadev** [hashicorp/terraform-provider-aws#24260]
### Terraform CLI and Terraform AWS Provider Version

```
`➜  eks git:(main) ✗ terraform -v       
Terraform v1.1.7
on darwin_arm64
+ provider registry.terraform.io/gavinbunney/kubectl v1.13.1
+ provider registry.terraform.io/hashicorp/aws v3.74.3
+ provider registry.terraform.io/hashicorp/helm v2.4.1
+ provider registry.terraform.io/hashicorp/kubernetes v2.7.1
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/hashicorp/tls v3.1.0`
```

### Affected Resource(s)

* aws_XXXXX

### Terraform Configuration Files

```hcl
resource "aws_fsx_lustre_file_system" "moodle_fsx_lustre_fs" {
  depends_on = [
    aws_cloudwatch_log_group.lustre_cloudwatch_log_group
  ]
  storage_capacity            = 1200
  subnet_ids                  = var.lustre_subnet_id
  deployment_type             = "PERSISTENT_2"
  per_unit_storage_throughput = 125
  log_configuration {
    destination = "/aws/fsx/lustre"
    level = ["WARN_ONLY", "FAILURE_ONLY", "ERROR_ONLY", "WARN_ERROR"]
  }
}
```

### Expected Behavior

Enable logging in CoudWatch.

### Actual Behavior

```
➜  eks git:(main) ✗ terraform validate
╷
│ Error: Unsupported block type
│ 
│   on lustre.tf line 14, in resource "aws_fsx_lustre_file_system" "moodle_fsx_lustre_fs":
│   14:   log_configuration {
│ 
│ Blocks of type "log_configuration" are not expected here.
```

Thank you!

**Comment by DrFaust92** [comment]
`log_configuration ` is available starting from v4.0.0+
https://github.com/hashicorp/terraform-provider-aws/pull/22935

**Comment by jesseadev** [comment]
Thank you @DrFaust92 . That did it.
